composer.json

```diff
@@ -32,6 +32,7 @@
         "stripe/stripe-php": "^9.1.0",
         "symfony/cache": "^6.1",
         "symfony/doctrine-bridge": "^6.1",
+        "symfony/http-client": "^6.2",
```

Comment:
Do we benefit from adding this on top of Guzzle? Do they share PSR interfaces these days?

Reply:
No, they don't share an interface - or if they do Symfony also implements its own interface. The Symfony NotComprimised validation rule doesn't work without it - it just triggers an error that tells you to install this. If we avoided this we'd have to use a different way to connect to the API. There's a lot on packagist but none else from anywhere with brand recognition comparable to Symfony. https://packagist.org/?query=haveibeenpowned